domains/client/domain-operator/src/aux_schema.rs

```diff
@@ -89,31 +91,37 @@ where
 
     let mut keys_to_delete = vec![];
 
-    // Delete ER that have comfirned long time ago, also see the comment of `PRUNING_DEPTH`
-    if let Some(delete_receipts_to) = oldest_unconfirmed_receipt_number
-        .map(|oldest_unconfirmed_receipt_number| {
-            oldest_unconfirmed_receipt_number.saturating_sub(One::one())
-        })
-        .and_then(|latest_confirmed_receipt_number| {
-            latest_confirmed_receipt_number
-                .saturated_into::<BlockNumber>()
-                .checked_sub(PRUNING_DEPTH)
-        })
+    if let Some(pruning_block_number) =
```

Comment:
Would this be clearer as `pruning_block_depth`? (Or count?)

When I see “block number”, it reads as a fixed or calculated number for a block, not a number *of* blocks (which is usually called a block count or block depth).

Reply:
not exactly, `pruning_block_number` is the block number at which we want prune the ER and other aux data

Reply:
we do not need to use different types since latest confirmed will be always unsigned and diff will always be >= 0

Reply:
Yes, I understand. I was talking about how I found a variable name confusing. The types were just an example of how other blockchains model the two separate concepts of "block height" and "block range".

Since we use the same type for both here, the only way we can separate them is by naming variables consistently.